keyboards/signum31/keymaps/default/generate_km.py

```diff
@@ -0,0 +1,104 @@
+from sortedcontainers import SortedDict
```

Comment:
This is an interesting script but I think it's out of scope for this PR. Could you pull this out for now and submit it as a separate PR? It should probably go into `/util`.

Reply:
The script is already part of qmk: `keyboards/signum/3_0/elitec/keymap/default/generate_km.py`. I've been working on it to reduce external dependencies and also to make it more generic. My feeling is that it needs lots more work to make it generic enough for other keyboards. If it's okay with you, I'll keep developing in the safety of the keymap folder before it gets exposed to the public. Maybe we can have an issue to collect ideas and round off the rough edges.